cluster-scope/base/user.openshift.io/groups/peribolos-as-a-service/group.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: peribolos-as-a-service
+users:
+  - tumido
+  - harshad16
```

Comment:
Feel free to add other team members: https://github.com/open-services-group/peribolos-as-a-service/blob/17c8e1fd9aed4c5dd75bdc0d2d2216c0b47ca67e/OWNERS#L6
and also people who are not in the list but are working on the project.

Reply:
Whoops. Too late. I think additional members can be added in subsequent PRs

Reply:
yeah no worries, lets see when that is needed.
thanks for all the work.